src/main/java/bowling/model/frame/FinalFrame.java

```diff
@@ -0,0 +1,55 @@
+package bowling.model.frame;
+
+import bowling.model.state.BonusOpen;
+import bowling.model.state.State;
+import bowling.model.state.finishedState.FinishedState;
+import bowling.model.state.finishedState.Miss;
+import bowling.model.state.finishedState.Spare;
+import bowling.model.state.finishedState.Strike;
+
+import java.util.Optional;
+
+public class FinalFrame extends Frame {
+
+    private Optional<State> previousState = Optional.empty();
+    private int countOfTry = 0;
+
+    public FinalFrame() {
+        frameNumber = FrameNumber.from(FrameNumber.MAX_FRAME_NUMBER);
+    }
+
+    @Override
+    public Frame bowling(int fallenPins) {
+        countOfTry++;
+        state = state.bowling(fallenPins);
+
+        if (isAbleToBonusFrame()) {
+            changeBonusFrame();
+            return this;
+        }
+
+        return this;
```

Comment:
다음과 같이 변경해도 괜찮을 거 같아요 :)
```suggestion
        if (isAbleToBonusFrame()) {
            changeBonusFrame();
        }

        return this;
```